### DISCUSSION TITLE
Streamlit Data Leak Error

### DISCUSSION THREAD

**Discussion by shinddayo** [teamalgoritma/community#892] [Q&A]
![image](https://github.com/user-attachments/assets/15508ad9-d05a-411e-be46-767dc3f66546)

apakah ada yang salah dengan Gsheets nya?
apa yang harus dilakukan kah?
untuk spreadsheet nya sudah di set access anyone with the link

**Comment by dwiknrd** [comment]
Dear Bapak [shinddayo](https://github.com/shinddayo),

Sehubungan dengan adanya pembaruan dari Streamlit baru-baru ini, versi default aplikasi yang terdeploy saat ini adalah streamlit==1.37.0. Hal ini mungkin menyebabkan error pada beberapa dependensi, terutama pada gsheet_connection. Oleh karena itu, kami menyarankan Bapak untuk memperbarui file requirements.txt dengan menyertakan versi Streamlit yang kompatibel, yaitu streamlit==1.36.0. Apakah bapak sudah memastikan pada requirements github repository versi streamlit sudah sesuai?
​
Demikian yang dapat kami sampaikan, apabila ada pertanyaan lebih lanjut silahkan ajukan dengan menjawab comment ini. Terima kasih.

Best Regards,
Team Algoritma
